data/sql/updates/pending_db_world/rev_1680005777631310400.sql

```diff
@@ -0,0 +1,225 @@
+-- Fire Bomb (Halaa)
+DELETE FROM `spell_custom_attr` WHERE `spell_id` = 31961;
+INSERT INTO `spell_custom_attr` (`spell_id`, `attributes`) VALUES
+(31961, 2048);
+
+-- NPC CREATURE
+SET @NPC := (SELECT MAX(`guid`) FROM `creature`);
```

Comment:
This is bad. Becausd I think the max GUID is unnecessarily large. Use tools like https://github.com/azerothcore/Unused-Guid-Search to find consecutive guid sequences instead. That way we can keep them as small as possible